File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Dãy số đẹp 2/DAYDEP2.py
s = 0
while True:
    x = int(input())
    s += x
    if x == 0:
        break

if s % 10 == 0:
    print("DEP")
elif s % 10 == 5:
    print("TRUNGBINH")
else:
    print("XAU")

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Dãy số đẹp 2/DAYDEP2.cpp
#include<bits/stdc++.h>
using namespace std;
int main (){
    int x;
    int s = 0;
    while(cin >> x){
        s += x;
        if(x == 0)
            break;
    }
    if(s % 10 == 0)
    {
        cout << "DEP";
        return 0;
    }if(s % 10 == 5)
    {
        cout << "TRUNGBINH";
        return 0;
    }
    cout << "XAU";
    return 0;
}